tests/conformance/common.go

```diff
@@ -53,6 +53,7 @@ import (
 	ss_local_file "github.com/dapr/components-contrib/secretstores/local/file"
 	s_cosmosdb "github.com/dapr/components-contrib/state/azure/cosmosdb"
 	s_mongodb "github.com/dapr/components-contrib/state/mongodb"
+	s_postgresql "github.com/dapr/components-contrib/state/postgresql"
```

Comment:
@beiwei30 can you fix the linter issue, it will block a merge? (Run `gofmt` on this) I think this may be because the entry is not in alphabetical order.

Reply:
done. @CodeMonkeyLeet pls. review again.